core/templates/dev/head/pages/teach.html

```diff
@@ -48,56 +48,51 @@
-                your creations with the community.
-              </p>
-            </div>
-            <div class="oppia-about-three-cols-layout oppia-about-three-cols-feature">
-              <img ng-src="<[getStaticImageUrl('/general/creator-feedback.png')]>">
-              <h3>Earn feedback</h3>
-              <p style="margin-top: -10px;">
-                to improve your exploration.
-              </p>
-            </div>
+          <div style="text-align: center; width: 100%;">
+            <em>Create high-quality lessons that help students learn difficult topics.</em>
           </div>
 
-          <hr>
+          <p>
+            You might know a student who's struggling with a topic at school.
+            They really want to learn it, but the lesson goes too fast, or the
+            textbook isn't easy to understand &mdash; and it's hard for them to
+            get the support they need. And it's not just this student: other
+            students, all over the world, are likely to struggle with the same
+            topics, again and again and again and again.
+          </p>
 
           <p>
-            How do pendulums work? Is light a particle or a wave? What is art?
-            Can you explain topics like these to a student who wants to learn
-            more? If so, you can use Oppia's unique technology to recreate 1-on-1
-            tutoring and reach students around the world by creating and sharing
-            a single piece of learning material.
+            In order to help address this, we're excited to announce the
+            <em>Teach with Oppia</em> program!
           </p>
 
           <p>
-            With Oppia, you can create <em>explorations</em> that help students
-            learn in a fun and creative way, using videos, images, and open-ended
-            questions. Students often have similar misconceptions, and Oppia
-            helps you organize these so that you can give targeted feedback to a
-            group of students at a time. By teaching with Oppia, you can
-            communicate with students around the world, understand different
-            points of view, and improve your skills in communication and empathy.
+            The <em>Teach with Oppia</em> program enables <strong>anyone</strong>
+            &mdash; especially students &mdash; to contribute to a collection
+            of free, accessible lessons on topics that learners find difficult.
+            The lessons use Oppia's unique technology to recreate 1-on-1
+            tutoring, and typically include clear, compelling and motivated
+            explanations, together with ample exercises that allow students to
+            gain proficiency with the subject. Students can then do these
+            lessons at their own pace, supported as much as possible by useful,
+            targeted feedback.
           </p>
+
+          <p>
+            Currently, participation in the program is <strong>free</strong>.
```

Comment:
Agreed. If the goal is to emphasize it's no cost to learners now, but may be in the future we could word is a bit differently:

``Participation in the program is currently <string>free</strong>, though new participants in the future may have to pay for this program.``

The above wording assumes that, once you join this program, you get to stay in it. If the program is a temporary participation, then that should be more clearly mentioned elsewhere on this page but the current wording indicating cost works just fine. The problem comes up when someone wants to join the program but worries it may cost them something in the future. That's not a good thought for someone to have, as it feels a bit sleazy. I think this wording should emphasize the following:

1. Current participation is free
2. Participants who join now will not be forced to pay anything in the future (if they program is continuous)
3. Participants who join in the future may have to pay something because this program requires Oppia Foundation resources and cannot be offered at no cost indefinitely